tests/playbooks/kubevirt_pvc.yml

```diff
@@ -2,22 +2,25 @@
 - name: Create PVCs and have CDI do something with them
   hosts: localhost
   connection: local
+  module_defaults:
+    group/k8s:
+      namespace: default
+    kubevirt_pvc:
+      size: 100Mi
```

Comment:
We set defaults but we ignore namespace. Why not to do the same?

Reply:
Corrected